packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -277,7 +279,13 @@ export interface BasicCrawlerOptions<
  *     handleRequestFunction: async ({ request }) => {
  *         // 'request' contains an instance of the Request class
  *         // Here we simply fetch the HTML of the page and store it to a dataset
- *         const { body } = await Actor.utils.requestAsBrowser(request);
+ *         const { body } = await gotScraping({
+ *             url: request.url,
+ *             method: request.method,
+ *             body: request.payload,
```

Comment:
we need to do something about this, so we support passing request instances directly. either at the SDK level, or in got-scraping. i guess it would be simpler to support `body` as an alias to payload in got-scraping, right?